src/main/resources/iiif.properties

```diff
@@ -7,6 +7,7 @@ record-api.path      = /record/v2
 # location where we check if full text pages are available or not
 fulltext-api.baseurl = https://iiif.europeana.eu
 fulltext-api.path    = /presentation/<collectionId>/<itemId>/annopage/<pageId>
+fulltext.summary.path = /presentation/<collectionId>/<itemId>/annopage/
```

Comment:
Can we explain the difference between the 2? What setting is used for what purposes?
Or would it be possible to use 1 settings for all uses?